scipy/stats/tests/test_morestats.py

```diff
@@ -979,6 +979,44 @@ def test_mood_bad_arg(self):
         # less than 3
         assert_raises(ValueError, stats.mood, [1], [])
 
+    def test_mood_alternative(self):
+
+        np.random.seed(0)
+        x = stats.norm.rvs(scale=0.75, size=100)
+        y = stats.norm.rvs(scale=1.25, size=100)
+
+        stat1, p1 = stats.mood(x, y, alternative='two-sided')
+        stat2, p2 = stats.mood(x, y, alternative='less')
+        stat3, p3 = stats.mood(x, y, alternative='greater')
+
+        assert stat1 == stat2 == stat3
+        assert_allclose(p1, 0, atol=1e-7)
+        assert_allclose(p2, p1/2)
+        assert_allclose(p3, 1 - p1/2)
+
+        with pytest.raises(ValueError, match="alternative must be..."):
+            stats.mood(x, y, alternative='ekki-ekki')
+
+    @pytest.mark.parametrize("alternative, expected",
+                             [('two-sided', (1.037127561496, 0.299676411857)),
+                              ('less', (1.0371275614961, 0.8501617940715)),
+                              ('greater', (1.037127561496, 0.1498382059285))])
+    def test_mood_alternative_against_R(self, alternative, expected):
```

Comment:
@tirthasheshpatel This test is _failing_ locally by a substantial amount. Could you check against R, too?

The existing tests (e.g. `test_mood`) claim to have compared against R `mood.test`, but the `mood.test` at https://rdrr.io/r/stats/mood.test.html does not produce the reference values. It's possible that the R test was changed? But it is unclear which is correct.

Reply:
Yes, I checked it. Sorry didn't see this comment as I was reviewing the changes!